.github/workflows/release-on-milestone-closed.yml

```diff
@@ -0,0 +1,57 @@
+# https://help.github.com/en/categories/automating-your-workflow-with-github-actions
```

Comment:
This release workflow is something I endorse to use, as it simplified releasing stuff to a single "close milestone" interaction in my own repos. If not wished, I can remove it, but here's the gist of it:

 1. branch has to be named `0.9.x` for this to be tagged as `0.9.0`
 2. need to configure the following secrets on the repository:
    * `GIT_AUTHOR_NAME`
    * `GIT_AUTHOR_EMAIL`
    * `SIGNING_SECRET_KEY` (contents of a GPG subkey)
    * `ORGANIZATION_ADMIN_TOKEN` (optional, but endorsed to get automation to switch default branches automatically)
 3. create a `0.9.0` milestone
 4. close milestone `0.9.0`
 5. observe the tooling do a release for you :-D 


More at https://github.com/laminas/automatic-releases#release-automation